utility/README.md

```diff
@@ -711,89 +1341,56 @@ _However, this logic ignores why consensus algorithms exist in the first place.
 
-We do see (and are working to mitigate) potential “bad luck” sessions where an application might receive less than an appropriate minimum relay amount.  This is not seen as a hard problem to overcome.  “Good Luck” sessions will also happen on occasion.  This is not seen as a problem at all.
+We do see (and are working to mitigate) potential “bad luck” sessions where an application might receive less than an appropriate minimum relay amount. This is not seen as a hard problem to overcome. “Good Luck” sessions will also happen on occasion. This is not seen as a problem at all.
 
 **The name of the actor ‘Fishermen’ has other meanings for other projects and should be changed.**
 
 Traditionally, Fishermen are monitoring actors in other Web3 projects. They allow minimum on-chain information by providing off-chain double-checks on the participants who make ‘claims’. Similar to Validators, though our flavor of Fishermen is different and unique, the core functionality of the actor remains the same.
 
-**Ability of Fishermen to replace servicers mid session makes it necessary that the size of the dispatch list for any given session cannot be fixed.**  (Open question)
+**Ability of Fishermen to replace servicers mid session makes it necessary that the size of the dispatch list for any given session cannot be fixed.** (Open question)
 
-Newly “recruited” nodes need the freedom to respond to application requests even though they are outside of the calculated list for that session.  We may be able to limit dispatch list growth to 2 X the computed size however since the Fisherman cannot replace more than the entire original list.  This opens a door for applications to double their effective dispatch list but does not allow abuse of relay usage.
+Newly “recruited” nodes need the freedom to respond to application requests even though they are outside of the calculated list for that session. We may be able to limit dispatch list growth to 2 X the computed size however since the Fisherman cannot replace more than the entire original list. This opens a door for applications to double their effective dispatch list but does not allow abuse of relay usage.
 
 **Memos might be important (open question)**
 
 If people want to keep memos (which are helpful in the case of traceability/auditability), a compromise between the full content of the memos and nothing might be the signature of the text that “would be” on the memo using the private key. Of course, this begs the question of how big is too big, and what’s the average and mean sizes for memos in our current chain.
```

Comment:
I have left some comments related to this 💙 Maybe we could use a uint64 where each number corresponds to some primary key on an off-chain database something simple maybe like

| MemoID      | URL |
| ----------- | ----------- |
| 1      | ipfs://...       |
| 2   | ipfs://...        |